### DISCUSSION TITLE
Easy way to play around with the app

### DISCUSSION THREAD

**Discussion by nrudnyk** [prose-im/prose-app-macos#122] [Q&A]
I'm wondering whether there's an easy way to poke around and dig deeper into implementation. It looks to be a very nice use-case of the TCA which I'd like to explore. Thanks!

**Comment by RemiBardon** [comment]
Hello, thank you for your interest and sorry for the delay.

Since our last contributions to this repository, we have been working on [prose-core-client](https://github.com/prose-im/prose-core-client), the core logic of the application. This broke central parts of the application which we are going to repair in the next months.

TCA has also changed a lot since our first implementation, and we will migrate the whole code base to make use of the amazing tools the library now provides.

Once that done, it should be a lot easier to play around with the app and its TCA core.

Please come back to us if you want further instructions once the app is back in a running state.
